freud/locality.pyx

```diff
@@ -774,9 +774,15 @@ cdef class _RawPoints(NeighborQuery):
 
 
 cdef class AABBQuery(NeighborQuery):
-    R"""Use an AABB tree to find neighbors.
+    R"""Use an Axis-Aligned Bounding Box (AABB) tree to find neighbors.
 
     Also available as ``freud.AABBQuery``.
+
+    Args:
+        box (:class:`freud.box.Box`):
+            Simulation box.
+        points (:class:`np.ndarray`):
```

Comment:
Could you copy this line in docstring from `NeighborQuery`, which also indicates the appropriate shape of the points array?

Reply:
It would also be great if you could propagate this change to `LinkCell`.